types/sinon-sandbox/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for sinon-sandbox 2.0.6
+// Project: https://github.com/wealthfront/sinon-sandbox
+// Definitions by: Jordan Harband <https://github.com/ljharb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { createSandbox } from 'sinon';
+
+declare const sandbox: ReturnType<typeof createSandbox>;
```

Comment:
can we simplify this?
```suggestion
import type { SinonSandbox } from 'sinon';

declare const sandbox: SinonSandbox;
```
(the outcome is same)

Reply:
Sure, i suppose? altho the outcome wouldn’t be the same in the future if the type, and return type of that function, ever differed.